pkg/apis/core/validation/validation.go

```diff
@@ -3598,6 +3665,32 @@ func validatePodConditions(conditions []core.PodCondition, fldPath *field.Path)
 	return allErrs
 }
 
+// ValidatePodEphemeralContainersUpdate tests that a user update to EphemeralContainers is valid.
+// newPod and oldPod must only differ in their EphemeralContainers.
+func ValidatePodEphemeralContainersUpdate(newPod, oldPod *core.Pod) field.ErrorList {
```

Comment:
I'm assuming the existing spec change detection in ValidatePodUpdate will keep these from changing on normal pod put/patch calls... probably worth adding an explicit test in TestValidatePodUpdate for to ensure a normal put can't do anything to ephemeral containers